esphome/components/deep_sleep/deep_sleep_component.cpp

```diff
@@ -15,13 +15,14 @@ void DeepSleepComponent::setup() {
   ESP_LOGCONFIG(TAG, "Running setup");
   global_has_deep_sleep = true;
 
-  const optional<uint32_t> run_duration = get_run_duration_();
+  const optional<uint32_t> run_duration = this->get_run_duration_();
   if (run_duration.has_value()) {
     ESP_LOGI(TAG, "Scheduling in %" PRIu32 " ms", *run_duration);
-    this->set_timeout(*run_duration, [this]() { this->begin_sleep(); });
+    this->set_timeout("DeepSleep_RunTime", *run_duration, [this]() { this->begin_sleep(); });
```

Comment:
Devs confirmed on discord that scheduled item name is per component so there is no need for DeepSleep_ prefix. 
```suggestion
    this->set_timeout("RunTime", *run_duration, [this]() { this->begin_sleep(); });
```

It would be good idea to make this name a constant in cpp and then reuse it. Makes it clearer and easier to follow in code.

Reply:
good idea.